edgelet/dps/src/lib.rs

```diff
@@ -22,4 +21,4 @@ pub use model::{
 };
 pub use registration::{DpsClient, DpsTokenSource};
 
-pub const DPS_API_VERSION: &str = "2018-11-01";
+pub const DPS_API_VERSION: &str = "2019-04-15";
```

Comment:
Have we confirmed this is the final version DPS is going to use?

Reply:
Yes.